src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figure = random.nextInt(5);
+        int firstParametr = random.nextInt(60) + 1;
+        int secondParametr = random.nextInt(70) + 1;
+        int thirdParametr = random.nextInt(80) + 1;

```

Comment:
You dont need this parameters in all figure classes (you need only one for Circle). Lets create 5 private methods getRandomCircle(), getRandomSquare(), ... and call them in switch-case construction, this will increase code readability. (Remember private methods should follow after public as helper ones) And thus we can generate as many parameters as we need